FormalConjectures/Util/Linters/AMSLinter.lean

```diff
@@ -0,0 +1,60 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+
+import FormalConjectures.Util.Attributes
+import Mathlib.Tactic.Lemma
+
+
+/-! # The AMS Linter
+
+The `AMSLinter` is a linter to aid with formatting contributions to
+the Formal Conjectures repository by ensuring that results in a file have
+the appropriate tags in order to distinguish between open/already solved
+problems and background results/sanity checks.
```

Comment:
Sounds a bit off, like it is left over from copying from some other linter...

Reply:
Fixed:)